core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -244,10 +244,10 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     }
   }
 
-  // Generate the random name for a temp folder in Tachyon
+  // Generate the random name for a temp folder in OffHeap
   // Add a timestamp as the suffix here to make it more safe
-  val tachyonFolderName = "spark-" + randomUUID.toString()
-  conf.set("spark.tachyonStore.folderName", tachyonFolderName)
+  val offHeapFolderName = "spark-" + randomUUID.toString()
```

Comment:
This is an API breaking change. We need to keep the old one around (as an alias), and deprecate it.